ui/app/components/ens-input.js

```diff
@@ -73,6 +72,7 @@ EnsInput.prototype.render = function () {
 EnsInput.prototype.componentDidMount = function () {
   const network = this.props.network
   const networkHasEnsSupport = getNetworkEnsSupport(network)
+  this.setState({ ensResolution: '0x0000000000000000000000000000000000000000' })
```

Comment:
When repeatedly using a long variable in a file, it's a good place to define a `const` at the top of the file. Avoid having to make sure you typed all those zeroes right each time.